src/Form.js

```diff
@@ -0,0 +1,254 @@
+                <label htmlFor="">Họ tên:</label>
+              </td>
+              <td>
+                <input
+                  type="text"
+                  value={name}
+                  onChange={this.handleChangeName} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Email:</label>
+              </td>
+              <td>
+                <input
+                  type="email"
+                  value = {email}
+                  onChange={this.handleChangeEmail} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Mật khẩu:</label>
+              </td>
+              <td>
+                <input
+                  type="password"
+                  value={pass}
+                  onChange={this.handleChangePass} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Avatar:</label>
+              </td>
+              <td>
+                <input
+                  type="file"
+                  value = {avatar}
+                  onChange={this.handleChangeAvatar} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Số điện thoại:</label>
+              </td>
+              <td>
+                <input
+                  type="number"
+                  value = {phone}
+                  onChange={this.handleChangePhone} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Tuổi:</label>
+              </td>
+              <td>
+                <select
+                  value = {age}
+                  onChange={this.handleChangeAge} >
+                  <option value="1">1 tuổi</option>
+                  <option value="2">2 tuổi</option>
+                  <option value="3">3 tuổi</option>
+                  <option value="4">4 tuổi</option>
+                </select >
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Giới tính:</label>
+              </td>
+              <td>
+                <input
+                  type="radio"
+                  value="male"
+                  checked={gender === "male"}
+                  onChange={this.handleChangeGender}/>Nam<br/>
+                <input
+                  type="radio"
+                  value="female"
+                  checked={gender === "female"}
+                  onChange={this.handleChangeGender}/> Nữ
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Sở thích:</label>
+              </td>
+              <td>
+                <input
+                  type="checkbox"
+                  value="soccer"
+                  defaultChecked={favorite.soccer}
+                  onClick={this.handleChangeSoccer}/> Đá bóng <br/>
```

Comment:
`onClick={this.handleChangeSoccer}`
Viết như thế này thì cái form có 1000 cái input checkbox thì tạo ra 1000 cái hàm `onChange` ư?